test/integration/relation.js

```diff
@@ -260,6 +280,117 @@ module.exports = function(Bookshelf) {
         equal(_knex.toString(), sql);
       });
 
+      it('should handle a hasOne relation with explicit foreignKeyTarget', function() {
+        const base = new Locale({ id: 1, isoCode: 'en' });
```

Comment:
I only used `const` because **eslint** was warning to do so, I even considered refactoring all the file. Why not use the **babel-register** compiler for mocha and have all code wrote in es6? If not, it would be great the fix the **eslint** configuration file to not warn about using `let` and `const`.